### PR TITLE
Add option --versioned to scrape verb

### PR DESCRIPTION
**Pull request by rhaschke**
This implements #124.

### CODE REVIEW

**Comment by rhaschke** [comment]
Note that this also includes #106 as I was in need of this feature as well.

File: src/wstool/multiproject_cli.py [hunk 1]
```diff
@@ -1327,9 +1327,12 @@ def cmd_scrape(self, target_path, argv, config=None):
             usage=usage,
             description=__MULTIPRO_CMD_DICT__["scrape"],
             epilog="See: http://www.ros.org/wiki/rosinstall for details\n")
-        parser.add_option("-y", "--confirm", dest="confirm", default='',
+        parser.add_option("-y", "--confirm", dest="confirm", default=False,
```
Comment (rsinnet) [inline src/wstool/multiproject_cli.py]: Could you please also update the bash completions if adding command-line options: https://github.com/vcstools/wstool/blob/master/completion/wstool-completion.bash

Comment (rhaschke) [reply src/wstool/multiproject_cli.py]: `scrape` wasn't in the bash completion list before, but I added it with all its options now :wink:

Comment (rsinnet) [reply src/wstool/multiproject_cli.py]: # :raised_hands:

**Comment by rhaschke** [comment]
Hi @wjwwood,
scanning my open PRs, I just noted that this one is not yet merged. I also noted the Travis failure. 
One failure was indeed due to a change I made: As I added a version column to the status output table, I had to adapt the corresponding unittest as well, which I now did.
However, the other Travis failures seem to be unrelated to this PR. The [master branch is failing](https://travis-ci.org/vcstools/wstool/builds/548236428) already as shown in this dummy PR (#136).

**Comment by rhaschke** [comment]
I fixed the unrelated Travis issues in #137.

**Comment by tkruse** [comment]
@romainreignier : It seems reasonable to assume it#s the same reason as for all other open PRs.

**Comment by romainreignier** [comment]
@tkruse If I understand well, this tool is kind of unmaintained, right?

**Comment by tkruse** [comment]
@romainreignier : yes, kind of

**Comment by Levi-Armstrong** [comment]
Is there any plans on merging this? It is a very useful feature when developing with a team and you have multiple repositories you are modifying that depend on each other, one can use this to create a new rosinstall file with the exact state of the workspace and provide it to another developer to quickly get there machine to the exact state you are developing from for testing.

**Comment by Levi-Armstrong** [comment]
Ping!

**Comment by dirk-thomas** [comment]
Closing since the repository is about to be archived: see #154.

**Pull request closed without merging**